### PR TITLE
Update API

### PR DESCRIPTION
**Pull request by JoshLove-msft**

### CODE REVIEW

File: sdk/eventgrid/Azure.Messaging.EventGrid/src/Generated/Models/AcsMessageReceivedEventData.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+
+namespace Azure.Messaging.EventGrid.SystemEvents
+{
+    /// <summary> Schema of the Data property of an EventGridEvent for a Microsoft.Communication.AdvancedMessageReceived event. </summary>
+    public partial class AcsMessageReceivedEventData : AcsMessageEventData
```
Comment (JoshLove-msft) [inline sdk/eventgrid/Azure.Messaging.EventGrid/src/Generated/Models/AcsMessageReceivedEventData.cs:13]: Should this actually be named AcsMessagesReceivedEventData to align with the Azure.Communication.Messages library?
(as suggested by @billwert)

**Review by tg-msft — APPROVED** [review]
The updates look good!

File: sdk/eventgrid/Azure.Messaging.EventGrid/src/Generated/Models/AcsMessageReceivedEventData.cs (hunk 1 above)
Comment (tg-msft) [reply sdk/eventgrid/Azure.Messaging.EventGrid/src/Generated/Models/AcsMessageReceivedEventData.cs:13]: I think `Message` reads better than `Messages` in most of the names here and the other ACS service prefixes in system events are all singular too.  I'd leave it as you have it here unless folks feel very strongly.

Comment (JoshLove-msft) [reply sdk/eventgrid/Azure.Messaging.EventGrid/src/Generated/Models/AcsMessageReceivedEventData.cs:13]: My understanding is that this is not used in the REST API anywhere.

**Review by jsquire — APPROVED** [review]
I have no opinion on the naming and am good with whatever direction is chosen.

**Pull request closed without merging**